.mergify.yml

```diff
@@ -9,42 +9,27 @@ queue_rules:
           - check-failure!=Integration tests
       - or:
           - label=bypass:integration
```

Comment:
Why not go all the way, and have a single `integration-result`?  Is there something I'm missing?

Then most of this logic can collapse into just:

```yaml
    - or:
          - label=bypass:integration
          - check-success=integration-result
          - check-neutral=integration-result
          - check-skipped=integration-result
```

Reply:
Yeah I guess. I had tried that before, but with the wrong `if` condition. This should work just as well

Reply:
Done. I also removed the `label=bypass:integration` because that label being applied will caused the result job to be skipped, and makes the logic for mergify simpler to realize the condition has no other way to succeed.